.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "node"
+  - "0.10"
```

Comment:
0.10 isn't supported by anyone any more... I reckon drop it so we can use more modern features (like ES6 in testing).